src/jit/codegencommon.cpp

```diff
@@ -4336,17 +4336,6 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
             }
         }
 
```

Comment:
I believe that the #ifdef on line 4017 shodul be changed to be ARM64 specific:


#ifdef _TARGET_ARM64_
            if (varDsc->lvIsMultiregStruct())
            {
...
#endif //  _TARGET_ARM64_